### PR TITLE
Mmann1123 extract tune2

### PR DESCRIPTION
**Pull request by mmann1123**
## Changes
- predict now also return Xy which can be used for cross validation and parameter tuning
- updated docs to reflect changes

### CODE REVIEW

File: geowombat/ml/classifiers.py [hunk 1]
```diff
@@ -344,6 +361,23 @@ def predict(
             .transpose("time", "band", "y", "x")
         )
 
+        # except AttributeError as e:
```
Comment (jgrss) [inline geowombat/ml/classifiers.py:364]: Any reason for keeping this?

File: geowombat/ml/classifiers.py [hunk 2]
```diff
@@ -269,7 +285,7 @@ def fit(
 
             clf.fit(Xna, y)
 
-        return X, clf
+            return X, (Xna, y), clf
```
Comment (jgrss) [inline geowombat/ml/classifiers.py:288]: I like this addition. Can `Xna` and `y` be combined into a single `DataArray`?

**Review by jgrss — APPROVED** [review]
👍

**Comment by jgrss** [comment]
@mmann1123 Since we don't have auto-change logging, do you want to update the readthedocs changelog to highlight some of the ML changes you've implemented?

File: geowombat/ml/classifiers.py (hunk 1 above)
Comment (mmann1123) [reply geowombat/ml/classifiers.py:364]: No I will move this to a feature branch. Trying to figure out how to pass best_model from trained pipeline.

**Comment by mmann1123** [comment]
Resolved comments

**Pull request merged**